zeppelin-server/src/main/java/org/apache/zeppelin/realm/ActiveDirectoryGroupRealm.java

```diff
@@ -202,6 +202,9 @@ protected AuthenticationInfo queryForAuthenticationInfo(
   }
 
   protected AuthenticationInfo buildAuthenticationInfo(String username, char[] password) {
+    if (this.principalSuffix != null && username.indexOf('@') > 1) {
+      username = username.split("@")[0];
```

Comment:
I'm assuming null/empty/whitespace user wouldn't reach here as it would have already failed from AD server, as not a valid user.
Null check is here https://github.com/apache/zeppelin/blob/master/zeppelin-server/src/main/java/org/apache/zeppelin/realm/ActiveDirectoryGroupRealm.java#L189